openmdao/core/problem.py

```diff
@@ -453,10 +455,16 @@ def check_partials(self, logger=None, comps=None, compact_print=False,
             Threshold value for relative error.  Errors about this value will have a '*' displayed
             next to them in output, making them easy to search for. Note at times there may be a
             significant relative error due to a minor absolute error.  Default is 1.0E-6.
-        global_options : dict
-            Dictionary of options that are used as default unless overridden by calling
-            set_check_partial_options on a component. Only 'form', 'step', 'step_calc', and
-            'method' can be specified in this way.
+        method : str
+            Method, 'fd' for finite difference or 'cs' for complex step.
```

Comment:
need to show default values in the docstring for `method`, `step`, `force_dense` and `suppress_output`?

Reply:
DONE. Thanks